src/nxdoc/nuxeo-server/administration/database-configuration/documentdb.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Amazon DocumentDB
+labels:
+    - database
+    - documentdb
+tree_item_index: 700
+
+---
+`DocumentDB` is compatible with `MongoDB`, therefore, it can be used by `Nuxeo` like a `MongoDB` backend,
+but there are limitations.
+
+{{#> callout type='warning' heading='elastic clusters are not supported'}}
+Only instance clusters are supported due to elastic clusters' limitations.
+{{/callout}}
+
+## Limitations
```

Comment:
Made minor changes to this section.

Reply:
we don't mention it that way but rather in your review comment.
This message in a notification doesn't hint at what "this section" refers to.

It could be
Added some rewording and spelling corrections for the "Limitations" section